Solutions/Corvus.Tenancy.Specs/Corvus.Tenancy.Specs.csproj

```diff
@@ -9,6 +9,7 @@
 
   <ItemGroup>
     <SpecFlowObsoleteCodeBehindFiles Remove="Features\CloudBlobContainer - Copy.feature.cs" />
+    <SpecFlowObsoleteCodeBehindFiles Remove="Features\CosmosContainer - Copy.feature.cs" />
```

Comment:
I've lost track, but are these the things that get left behind because of old legacy behaviour that we should delete, or are they a subtly different thing that has recently started appearing due to changes in SpecFlow?

They don't _look_ like a thing we want. Not with that '- Copy' in there.

Reply:
No - these are clearly "stuffed in there by the tooling and can be deleted"